Dockerfile

```diff
@@ -4,8 +4,8 @@
 #
 # Separating the builder and runtime image allows the runtime image to be
 # considerably smaller because it doesn't need to have Golang installed.
-ARG BUILDER_IMAGE=docker.io/golang:1.22.2-alpine3.18
-ARG RUNTIME_IMAGE=docker.io/alpine:3.19.1
+ARG BUILDER_IMAGE=docker.io/golang:1.22.2-alpine3.19
+ARG RUNTIME_IMAGE=docker.io/alpine:3.19
```

Comment:
Quoting from the issue underlying this PR:
> Upgrade all instances of Alpine Linux to the latest supported: 3.19.1

Here, the applied modification does not seem aligned with what has been indicated as a fix in the issue.